### PR TITLE
Revert unnecessary changes

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

**Comment by jkotas** [comment]
@dotnet/nativeaot-llvm

File: src/coreclr/nativeaot/CMakeLists.txt [hunk 1]
```diff
@@ -30,10 +30,6 @@ if(CLR_CMAKE_HOST_UNIX)
   endif(CLR_CMAKE_TARGET_ARCH_AMD64 OR CLR_CMAKE_TARGET_ARCH_I386)
 endif (CLR_CMAKE_HOST_UNIX)
 
-if(CLR_CMAKE_TARGET_ARCH_WASM)
```
Comment (jkotas) [inline src/coreclr/nativeaot/CMakeLists.txt:33]: Moved to src/coreclr/nativeaot/Runtime/CMakeLists.txt

File: eng/pipelines/installer/jobs/build-job.yml [hunk 2]
```diff
@@ -1,384 +0,0 @@
-parameters:
```
Comment (SingleAccretion) [inline eng/pipelines/installer/jobs/build-job.yml:1]: Here's a list of other files that look to be merge leftovers I made with `git diff 9b57a265c7e --stat=200 --diff-filter A`:
```
 eng/pipelines/installer/jobs/build-job.yml                                                                                   |  384 ++++++
 eng/pipelines/runtimelab/coreclr-tests.yml                                                                                   |   41 +
 src/coreclr/Corehost.Static/config.h                                                                                         |    7 +
 src/coreclr/inc/clr_std/utility                                                                                              |  269 ++++
 src/coreclr/nativeaot/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/CallConverterThunk.cs                        | 1414 ++++++++++++++++++++
 src/coreclr/tools/Common/JitInterface/CorInfoImpl.Intrinsics.cs                                                              |  156 +++
 src/coreclr/tools/aot/ILCompiler.LLVM/libLLVMdep.depproj                                                                     |   26 +
 src/libraries/Native/Unix/CMakeLists.txt
 src/libraries/System.Linq.Expressions/src/MatchingRefApiCompatBaseline.txt                                                   |   21 +
 src/coreclr/System.Private.CoreLib/src/System/Exception.CoreCLR.LLVM.cs                                                      |   42 +
```

File: src/coreclr/nativeaot/Runtime/CommonMacros.h [hunk 3]
```diff
@@ -143,11 +143,7 @@ inline bool IS_ALIGNED(T* val, uintptr_t alignment);
 
 #define DATA_ALIGNMENT  8
```
Comment (SingleAccretion) [inline src/coreclr/nativeaot/Runtime/CommonMacros.h:144]: By the way, `DATA_ALIGNMENT` looks like dead code to me.

File: eng/pipelines/installer/jobs/build-job.yml (hunk 2 above)
Comment (jkotas) [reply eng/pipelines/installer/jobs/build-job.yml:1]: Nice!

File: src/coreclr/nativeaot/Runtime/CommonMacros.h (hunk 3 above)
Comment (jkotas) [reply src/coreclr/nativeaot/Runtime/CommonMacros.h:144]: Yes, it is dead code. Deleting in https://github.com/dotnet/runtime/pull/101655

**Review by SingleAccretion — APPROVED** [review]
LGTM, thank you!

**Pull request merged**